elastic/kibana
# [AO] Add threshold information to the metric threshold alert details page

Closes #153740, closes #153833, closes #155593

This PR adds threshold information and rule name to the metric threshold alert details page.

![image](https://user-images.githubusercontent.com/12370520/233968325-8a66166b-2534-4b9b-9054-9085270db5f6.png)


## 🧪 How to test
- Add xpack.observability.unsafe.alertDetails.metrics.enabled: true to the Kibana config
- Create a metric threshold rule with multiple conditions that generates an alert
- Go to the alert details page and check threshold information
- Click on the rule link; it should send you to the rule page
